dev-libs/mdns/files/mdns-1.4.2-timeval.patch

```diff
@@ -0,0 +1,12 @@
+https://github.com/mjansson/mdns/pull/69
+
+--- a/mdns.c
```

Comment:
Please send this upstream, but lgtm.

Reply:
Please see the comment on top of this file

Reply:
Sorry, not sure how I missed it!